unit/c2goto/library/builtin_libs.test.cpp

```diff
@@ -0,0 +1,43 @@
+/*******************************************************************
+ Module: Builtin Libs unit test
+
+ Author: Rafael Sá Menezes
+
+ Date: December 2020
+
+ Test Plan:
+   - Functions (Sync)
+   - Functions (Async - Pthread)
+ \*******************************************************************/
+
+#define BOOST_TEST_MODULE "Builtin Models"
+
+#include <buitin_libs.hs>
+#include <boost/test/included/unit_test.hpp>
+
+
+void __ESBMC_atomic_begin() {}
+void __ESBMC_atomic_end() {}
+
+
+namespace utf = boost::unit_test;
+
+#define sync_fetch_generator(TYPE, OPERATOR) \
+    {int dest = 10;                     \
+    TYPE value = 5;                          \
+    int fetch = __builtin_esbmc_sync_fetch_and_##OPERATOR(&dest, value); \
+    BOOST_TEST(dest == 15);                  \
+    BOOST_TEST(fetch == 10);}
+
```

Comment:
apply clang-format to this file (it should format this macro).

Our CI should've complained about it.

Reply:
Ok, I think the issue is that test-cases are not triggering the clang-format

Reply:
Applyed clang-format and fixed it on CI